docs/expo-dict.json

```
@@ -1,7 +1,7 @@
 {
   "amazon appstore": "Amazon Appstore",
   "android emulator": "Android Emulator",
-  "app store": "App Store",
+  "apple app store": "Apple App Store",
```

```suggestion
  "app store": "App Store",
  "apple app store": "Apple App Store",
```

Keeping the terms separated should improve the checks quality, since IIRC case-lint ignores not fully matching terms, but you might want to verify that bechaviour.

---

Thanks for the tip! I'd still remove "App Store" because at times, we do use "app store" or "app stores" as a general term to signify the message.

---

The specific issue here which isn't solve was to rename the page "App store submissions" since the case-lint fails on the term "App store" and wants it to be "App Store".

---

I'm going to go ahead and merge this PR without this suggestion, thanks for approving it!

---

Make sense then, thank you for the research on that and context! 👍